server/migrations/20170303151657-create-reminders.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Reminders', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      status: {
```

Comment:
let's rename this to "state" to match the state field in other tables.  Also add a defaultValue of zero.